public/stylizer.js

```diff
@@ -61,5 +63,5 @@ addStyle("Arduino Light", "arduino-light.css");
 addStyle("Googlecode", "googlecode.css");
 
 
-document.body.appendChild(metaViewport);
-document.body.insertBefore(dropdown, document.body.firstChild);
+selector.appendChild(metaViewport);
+selector.insertBefore(dropdown, selector.firstChild);
```

Comment:
This (including line 25) will add these html tags to the dropdown!
![image](https://user-images.githubusercontent.com/32071671/66640422-4192fb80-ec19-11e9-8ae7-a67d85e1a0c5.png)
It should not do that.

I propose splitting dropdown in a seperate `.js` file and remove the logic from `stylizer.js`, then add the script tags o only for the new `dropdown.js` in `index.html`, but both when calling `writeWithHighlight`.